docs/01-Configuration.md

```diff
@@ -0,0 +1,327 @@
+
+```
+[ethereum]
+gas_price_multiplier = 1.0
+```
+
+#### `key_derivation_path`
+
+Type: string
+
+The path used to derive Etherum addresses from your private keys.
+
+```
+[ethereum]
+key_derivation_path = "m/44'/60'/0'/0/0"
+```
+
+#### `rpc`
+
+Type: string
+
+The node endpoint to be used for Ethereum interactions
+
+```
+[ethereum]
+rpc = "http://localhost:8545"
+```
+
+### `[gravity]` table
+
+Configuration related to the gravity bridge
+
+#### `contract`
+
+Type: string
+
+The address of the gravity contract on Ethereum
+
+```
+[gravity]
+contract = "0x0000000000000000000000000000000000000000"
+```
+
+#### `fees_denom`
+
+Type: string
+
+The denomination in which fees will be paid to relayers. This value must be a denom supported by the configured chain (whatever chain the node at `cosmos.grpc` is validating on).
+
+```
+[gravity]
+fees_denom = "usomm"
+```
+
+### `[keys]` table
+
+Keys config
+
+#### `delegate_key`
+
+Type: string
+
+The name of the key in `keystore` used for delegate signing by both Steward and the Orchestrator. The key name is the name of the key file *without the file extension*.
+
+```
+[keys]
+delegate_key = ""
+```
+
+### [metrics] table
+
+Config related to the Orchestrator metrics server
+
+#### `listen_addr`
+
+Type: string
+
+The server endpoint for monitoring Orchestrator metrics
+
+```
+[metrics]
+listen_addr = "127.0.0.1:3000"
+```
+
+### [server] table
+
+Config related to the Steward server
+
+#### `address`
+
+Type: string
+
+The IP address at which the Steward server will run
+
+```
+[server]
+address = "0.0.0.0"
+```
+
+#### `client_ca_cert_path`
+
+Type: string
+
+The path to the trusted CA used to sign the Strategy Provider's client certificate
+
+> :warning: Please leave this value unset for now as it defaults to the Peggy JV CA internally
+
+```
+[server]
+client_ca_cert_path = ""
+```
+
+#### `port`
+
+Type: integer
+
+The port on which the Steward server will be hosted
+
+```
+[server]
+port = 5734
+```
+
+#### `server_cert_path`
+
+Type: string
+
+The path to your signed Steward server certificate
+
+```
+[server]
+server_cert_path = ""
+```
+
+#### `server_key_path`
+
+Type: string
+
+The path to the PKCS8-formatted key used to generate the server certificate
+
+```
+[server]
+server_key_path = ""
+```
+
+## Complete Example config.toml
+
+This example will not work as is, you'll need to supply your own values.
+
+```toml
+keystore = "/some/path/keystore"
+
+[cosmos]
+gas_adjustment = 1.0
+grpc = "https://127.0.0.1:9090"
+key_derivation_path = "m/44'/118'/0'/0/0"
+prefix = "somm"
+msg_batch_size = 5
+
+[cosmos.gas_price]
+# Please keep amount set to 0.0 at this time
+amount = 0.0
+denom = "usomm"
+
+[ethereum]
+blocks_to_search = 5000
+gas_price_multiplier = 1.0
+key_derivation_path = "m/44'/60'/0'/0/0"
+rpc = "http://localhost:8545"
+
+[gravity]
+contract = "0x0000000000000000000000000000000000000000"
+fees_denom = "stake"
+
+[keys]
+delegate_key = ""
```

Comment:
Since this is an example config, let's put an example name in here, e.g. "key_name" or something.